esx_service/vmdk_ops_test.py

```diff
@@ -1082,6 +1084,87 @@ def test_vmdkops_on_different_tenants(self):
         self.assertEqual("tenant2_vol1", result[0]['Name'])
         self.assertEqual("tenant2_vol2", result[1]['Name'])
         self.assertEqual("tenant2_vol3", result[2]['Name'])
+    
+    @unittest.skipIf(len(vmdk_utils.get_datastores()) < 2,
+                     "second datastore is not found - skipping this test")
```

Comment:
As we discussed yesterday, I will keep this code as it is.
I will file a new issue to track:
 -- testbed in CI need to make sure it have 2 datastores  in order to run this test successfully
-- contributing.md need to be updated to let user know they need to have 2 datastores to run the test